content/en/docs/concepts/storage/volumes.md

```diff
@@ -965,7 +965,7 @@ If vSphere CSI Driver is not installed volume operations can not be performed on
 
 You must run vSphere 7.0u2 or later in order to migrate to the vSphere CSI driver.
 
-If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, consult
+If you are running a version of Kubernetes other than v{{< skew currentPatchVersion >}}, consult
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.